### PR TITLE
 Implement partially Req 3.4 PCIDSS

### PR DESCRIPTION
**Pull request by sherine-k**
#### Description:

- This PR mainly contains implementation of requirement 3.4.1
- It also answers requirements 3.1->3.4 of the PCIDSS certification

### CODE REVIEW

**Comment by JAORMX** [comment]
/retest

Timed out setting IdP. This is unrelated to the rules that were added here.

**Comment by jhrozek** [comment]
This seems related:
```
     helpers.go:804: Result - Name: e2e-pci-dss-fips-mode-enabled - Status: FAIL - Severity: high
    helpers.go:807: E2E-FAILURE: The expected result for the fips_mode_enabled rule didn't match. Expected 'PASS', Got 'FAIL' 
```

File: controls/pcidss_ocp4.yml [hunk 1]
```diff
@@ -774,7 +810,8 @@ controls:
     - base
     notes: ''
     status: pending
-    rules: []
+    rules: 
+    - fips_mode_enabled
```
Comment (JAORMX) [inline controls/pcidss_ocp4.yml:814]: This makes FIPS mandatory for PCI-DSS

File: applications/openshift/integrity/crypto/ebs_encryption_enabled/rule.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+prodtype: ocp4
+
+title: Ensure that FIPS mode is enabled on worker nodes
```
Comment (JAORMX) [inline applications/openshift/integrity/crypto/ebs_encryption_enabled/rule.yml:3]: @sherine-k remember to update the title and description here.

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: applications/openshift/integrity/crypto/machine_volume_encrypted/rule.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+prodtype: ocp4
+
+title: Ensure that FIPS mode is enabled on worker nodes
```
Comment (JAORMX) [inline applications/openshift/integrity/crypto/machine_volume_encrypted/rule.yml:3]: This still refers to FIPS

**Review by JAORMX — COMMENTED** [review]
A minor comment left. Gotta change the title and description of one rule still. The rest looks good to me.

**Comment by JAORMX** [comment]
/retest

Timed out applying remediations

**Comment by JAORMX** [comment]
```
cce CCE-85861-3 is included in files: 
 - applications/openshift/integrity/crypto/storageclass_encryption_enabled/rule.yml
 - applications/openshift/integrity/crypto/luks_enabled_on_all_nodes/rule.yml
```

**Review by JAORMX — APPROVED** [review]

**Comment by JAORMX** [comment]
/retest

Timed out remediating

**Pull request merged**